Simplified/en.lproj/Localizable.strings

```diff
@@ -270,3 +270,9 @@
 
 // Announcement
 "Announcement" = "Announcement";
+
+// Library Finder
+"Find a Library" = "Find a Library";
+"LibraryListUpdateFailed" = "Failed to update library list";
```

Comment:
can you please change this key and just use the actual English value as key, like the others? This will break for languages with incomplete localizations otherwise.